vm/src/stdlib/mod.rs

```diff
@@ -123,6 +124,9 @@ pub fn get_module_inits() -> HashMap<String, StdlibInitFunc> {
     #[cfg(any(unix, windows, target_os = "wasi"))]
     modules.insert(os::MODULE_NAME.to_owned(), Box::new(os::make_module));
 
+    #[cfg(any(unix, windows, target_os = "wasi"))]
+    modules.insert("_ctypes".to_owned(), Box::new(ctypes::make_module));
```

Comment:
instead of duplicating same cfg, it can be grouped like:
```rust
    #[cfg(any(unix, windows, target_os = "wasi"))]
    {
        modules.insert(os::MODULE_NAME.to_owned(), Box::new(os::make_module));
        modules.insert("_ctypes".to_owned(), Box::new(ctypes::make_module));
    }
```

Reply:
Although I don't know if ctypes would be available on wasi, I don't think it has dll functionality yet

Reply:
Would libffi work?

Reply:
No, wasm doesn't really have any sort of support for loading another wasm module at runtime